Marlin/Marlin_main.cpp

```diff
@@ -5315,12 +5312,12 @@ void home_all_axes() { gcode_G28(true); }
           }
         }
         if (_7p_intermed_points) // average intermediates to tower and opposites
-          for (uint8_t axis = 1; axis <= 11; axis += 2)
+          for (uint8_t axis = 1; axis < 13; axis += 2)
```

Comment:
< 13 is full circle even if it stops at 11; the use of <13 and mix with <=11 is confusing